multimodal-understanding/repeatable-patterns/10-tool-calling-with-converse/10_tool_calling_with_converse.ipynb

```diff
@@ -0,0 +1,168 @@
+{
```

Comment:
<p>The notebook looks good but it can be better and more readable if you can run it through a lint. Some lines are too long.  </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/aws-samples/amazon-nova-samples/pull/17/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='0'/>